src/EditorFeatures/VisualBasicTest/InitializeParameter/AddParameterCheckTests.vb

```diff
@@ -268,19 +268,118 @@ end class")
         End Function
 
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)>
-        Public Async Function TestNotOnLambdaParameter() As Task
-            Await TestMissingInRegularAndScriptAsync(
+        Public Async Function TestOnMultiLineSubLambdaParameter() As Task
+            Await TestInRegularAndScript1Async(
+"
+Imports System
+
+class C
+    public sub new()
+        dim f = sub ([||]s as string)
+                end sub
+    end sub
+end class",
+"
+Imports System
+
+class C
+    public sub new()
+        dim f = sub (s as string)
+
```

Comment:
All these blank lines in VB... the VB syntax generator does that for some reason, I don't know why or what the purpose of that is so I'm not changing it. It's in `VisualBasicSyntaxGenerator.WithStatements` but if you look at `CSharpSyntaxGenerator.WithStatements`, there's no code like that.

Reply:
yeah... that's not good.   we'd def want to fix this.  or at least open a bug here.  we don't do this for normal sub/function methods right?

Reply:
yes this only happens with lambdas

Reply:
so i checked again and it does create blank lines between the statements in methods, but not before the first one... in lambdas it also creates a blank line before the first statement

Reply:
I was suspicious this might have been caused by the call to Isolate (which calls PreserveTrivia) in VisualBasicSyntaxGenerator.WithStatements - there's no such thing in CSharpSyntaxGenerator, but that's not the case, I get same behavior if I directly call WithStatementsInternal. It must be some sort of formatting issue.

Reply:
@CyrusNajmabadi I identified the issue. It's in [ElasticTriviaFormattingRule.TopLevelStatement](http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic.Workspaces/Formatting/Rules/ElasticTriviaFormattingRule.vb,370). It's a one-line fix to add `TypeOf statement Is LambdaHeaderSyntax`. Should I include it in this PR?

Reply:
I'd have no problem with that.